addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -146,7 +146,10 @@ def migration_invoice_moves(env):
         write_date) = (
         ai.message_main_attachment_id, ai.access_token,
         COALESCE(ai.number, ai.move_name, am.name), COALESCE(ai.date, am.date),
-        ai.name, ai.comment, ai.type, ai.journal_id, ai.company_id,
+        CASE WHEN ai.type IN ('in_invoice', 'in_refund') THEN ai.reference ELSE ai.name END,
+        CASE WHEN ai.type IN ('in_invoice', 'in_refund') THEN COALESCE(ai.comment, ai.name)
```

Comment:
Why do you need to provide alternative "comments"?

Reply:
It's not "needed" strictly, but I observed in my BD, in old invoices of type '`in_*`' that the ai.name was used as a comment (for example "Wrong currency" or "Wrong payment mode"). Thus, having this info in the invoice_payment_ref doesn't make much sense, and I thought putting that info in the narration field makes more sense.

Reply:
I think that case is only in refunds

Reply:
You are right, I checked it.

Reply:
What do you propose?

Reply:
Split the case more for each type (`in_refund`, `in_invoice`, `out_refund`, `out_invoice`)

Reply:
done. can you check again?